themes/bootstrap3/js/preview.js

```diff
@@ -43,7 +43,7 @@ function applyPreviewUrl($link, url) {
 
     // Update associated record thumbnail, if any:
   $link.parents('.result,.record')
-        .find('.recordcover').parents('a').attr('href', url);
+        .find('.recordcover[data-linkpreview="true"]').parents('a').attr('href', url);
```

Comment:
Good point regarding selectors -- nothing protects us completely against side effects. :) In any case, I'll go ahead and merge!